heutagogy/auth.py

```diff
@@ -1,32 +1,48 @@
 from heutagogy import app
-import flask_login
-
-
-login_manager = flask_login.LoginManager()
-login_manager.init_app(app)
-
-
-class User(flask_login.UserMixin):
-    pass
-
-
-@login_manager.unauthorized_handler
-def unauthorized_handler():
-    return {'error': 'Unauthorized'}, 401
-
+from flask_jwt import JWT
+from werkzeug.security import safe_str_cmp
+
+
+class User(object):
+    def __init__(self, id, username, password):
+        self.id = id
+        self.username = username
+        self.password = password
+
+    def __str__(self):
+        return "User(id='%s')" % self.id
```

Comment:
I've just grabbed this code from [Flask-JWT](https://pythonhosted.org/Flask-JWT) home page.
(but you are right, username may be there as well)